README.md

```diff
@@ -27,7 +27,7 @@ commits and tags:
 * works without any VCS, but happily reads tag information from and writes
   commits and tags to Git and Mercurial if available
 * just handles text files, so it's not specific to any programming language
-* supports Python2, Python3 and Pypy
```

Comment:
Let's add a separate notice that if folks want to use Python2 they can target >1.0.0

Reply:
If people have modern pip 9+, the `python_requires='>=3.5'` metadata will ensure they get the last supported version.

So in https://github.com/c4urself/bump2version/pull/108/commits/32933285b70a6c24fc77106ac8a532fa952f4de5 I added:

> If you want to use Python 2, use `pip>=9` and you'll get the last supported version, or pin `bump2version<1`.

How's that sound? (And I assume you meant `less than` 1.0.0 :)

Reply:
yep, my bad :)